fecgov/fec-cms
# Search.gov env var update

## Summary (required)
Make DIGITALGOV_DRAWER_HANDLE into an env var.
Update other search.gov vars in CF User Provided Services (see issue for specifics)

- Resolves FEC_Accounts issue 271
https://github.com/fecgov/fec-accounts/issues/271

## Impacted areas of the application
modified:  fec/settings/base.py
User provided services on all four cloud environments
 
- [x] stage

## How to test
Since these env vars are not used in cms, there is nothing test there. However one could  log into the dashboard ([see the bottom of the issue for URL](https://github.com/fecgov/fec-accounts/issues/271)) and confirm the digitalgov vars there matches what is in CF UPS.
____
